react/react1/week2/vite-project/src/TodoList/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,25 @@
+import { React, useState } from "react";
+
+export function TodoItem(props) {
+  const [done, setDone] = useState(false);
```

Comment:
Maybe let's have the done state of the todo inside the todo items. That way, the state of the todos are not dependant of the state of the component